presto-product-tests/conf/presto/etc/multinode-worker.properties

```diff
@@ -11,6 +11,6 @@ node.environment=test
 coordinator=false
 http-server.http.port=8081
 query.max-memory=1GB
-query.max-memory-per-node=512MB
+query.max-memory-per-node=768MB
```

Comment:
`sql_tests.testcases.hive_tpch.q18` failed because exceeding node query memory limit. However it was already on the edge with top consuming operators:
```
Query exceeded per-node user memory limit of 512MB [Allocated: 511.83MB, Delta: 1.69MB,
Top Consumers: {HashBuilderOperator=363.73MB, HashAggregationOperator=144.22MB, PartitionedOutputOperator=42.01MB}]
```
`HashSemiJoinOperator` is not even on top consumers list

Reply:
Was this change related to the PR?

Reply:
Memory accounting was improved as part of this PR (and other work processor PRs). Also first page is hold until build side is ready. It tipped already borderline mem limit.